pkg/kopia/cli/internal/opts/cache_opts.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2024 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opts
+
+import (
+	"strconv"
+
+	"github.com/kanisterio/kanister/pkg/kopia/cli/args"
+	"github.com/kanisterio/safecli/command"
+)
+
+const (
+	defaultCacheDirectory = "/tmp/kopia-cache"
+)
+
+// CacheDirectory creates a new cache directory option with a given directory.
+// If the directory is empty, the default cache directory is used.
+func CacheDirectory(dir string) command.Applier {
+	if dir == "" {
+		dir = defaultCacheDirectory
+	}
+	return command.NewOptionWithArgument("--cache-directory", dir)
```

Comment:
I prefer to leave it as a constant, since using the `defaultCacheDirectory` here improves code readability and maintainability by centralizing the default cache directory path. It helps to make any future updates simpler and reducing the risk of typos.